### ISSUE TITLE
Prediction accuracy

### ISSUE THREAD

**Issue by Klara8A** [weecology/DeepForest-pytorch#80]
Hi!
I started using DeepForest in April when it was still running with tensorflow and now I have started testing it with the new version based on pytorch. I have noticed that the predictions are different. Being those of the new version less accurate and with more "errors". I have tried  it with different patch sizes, and different iou. 
So  I was wondering if this is due to the changes that have been made and if this is going to improve in the following releases ?

I have not yet tested it after training, but in general I feel that the results of the untrained model are less accurate than before.

**Comment by bw4sz** [comment]
A couple users had reported similar issues. I haven't been able to reproduce at all, the benchmark score is within a few percent, see the other issues. Can you paste images (before and after) and the code you used? I'm considering training a new model, but I need to understand where differences might be, since the training data is identical.

**Comment by bw4sz** [comment]
also, can we move this to the deepforest repo, this repo is deprecated as we merge the two.
